tests/src/python/test_qgsserver_accesscontrol_wms_getmap_postgres.py

```diff
@@ -0,0 +1,260 @@
+# -*- coding: utf-8 -*-
+"""QGIS Unit tests for QgsServer.
+
+From build dir, run: ctest -R PyQgsServerAccessControlWMSGetMapPG -V
+
+.. note:: This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+"""
+__author__ = 'Stephane Brunner'
+__date__ = '28/08/2015'
+__copyright__ = 'Copyright 2015, The QGIS Project'
+
+print('CTEST_FULL_OUTPUT')
```

Comment:
What is this for?

Reply:
It comes from `test_qgsserver_accesscontrol_wms_getfeatureinfo_postgres.py`. I am removing it.

Reply:
I have removed it and from other qgis server test files.